gridcomps/History/MAPL_HistoryGridComp.F90

```diff
@@ -3861,9 +3866,9 @@ subroutine MAPL_GradsCtlWrite ( clock, state,list,fname,expid,expdsc,output_grid
             call MAPL_GridGet(pgrid,globalCellCountPerDim=dims,_RC)
             IM = dims(1)
             JM = dims(2)
-            DLON   =  360._REAL64/real(IM)
+            DLON   =  360._REAL64/float(IM)
```

Comment:
Also completely unnecessary.   "IM" will be promoted to 64 bit precision when the division is processed.

Reply:
```suggestion
            DLON   =  360._REAL64/IM
```

Reply:
Thanks, @tclune.  I had been staring at this change before I created the PR; I could not figure out why it happened.  I had thought NAG asked me to change it, but I donot remember at all.  

Now I figured out, it is because I did not merge this file from my working repository to the develop branch, instead I copied the MAPL_HistoryGridComp.F90 from my directory to here. So I missed the modification introduced by Matt (Aug 31 16:20:40 2023).  I will use a merge tool to do this. 

I agree  Fortran default promotion.  360._REAL64/IM  will do the work.   Other modifications 360._REAL64/ real(IM, kind=REAL64) is redundant.  I will make the changes.